app/views/events/index.html.erb

```diff
@@ -16,6 +17,7 @@
           <td><%= event.name %></td>
           <td><%= event.duration %></td>
           <td><%= event.started_at %></td>
+          <td><%= event.event_status %></td>
```

Comment:
Please change the name of method. You have two times event and this is not needed. It will be good to just have `status`